Doc/library/os.rst

```diff
@@ -1087,9 +1087,9 @@ or `the MSDN <https://msdn.microsoft.com/en-us/library/z0kc8e3z.aspx>`_ on Windo
    .. versionadded:: 3.3
 
 
-.. function:: pwrite(fd, str, offset)
+.. function:: pwrite(fd, buffer, offset)
 
-   Write *bytestring* to a file descriptor, *fd*, from *offset*,
+   Write *buffer* to a file descriptor, *fd*, from *offset*,
```

Comment:
I prefer os.write() phrasing: "Write the bytestring in *str* to file descriptor *fd* (...), so please keep the parameter name "str".

Reply:
Actually, I do not agree with this statement. Maybe it's just a wording, but for me, speaking about _str_(ing) suppose we are dealing with NULL terminated buffer.

Finally, if we decide to go in favour of keeping _str_, then should the following be changed as well ?
```
.. function:: pread(fd, buffersize, offset)

   Read from a file descriptor, *fd*, at a position of *offset*. It will read up
   to *buffersize* number of bytes. The file offset remains unchanged.

   Availability: Unix.
```

Reply:
> Actually, I do not agree with this statement. Maybe it's just a wording, but for me, speaking about str(ing) suppose we are dealing with NULL terminated buffer.

It's the documentation of Python. In Python, it's perfectly fine to have NUL bytes in bytes strings and NUL characters in str strings. If you dislike "str" parameter name, please update the write() documentation as well.